lib/unleash/context.rb

```diff
@@ -28,7 +28,9 @@ def get_by_name(name)
       if ATTRS.include? normalized_name
         self.send(normalized_name)
       else
-        self.properties.fetch(normalized_name)
+        self.properties.fetch(normalized_name) {
+          || self.properties.fetch(name.to_sym, nil)
+        }
```

Comment:
This change has side-effects with current behaviour (which is tested explicitly in #71). 

IIRC this change sorta makes sense, (though I would have prefered a different syntax here).

```suggestion
        self.properties.fetch(normalized_name, nil) || self.properties.fetch(name.to_sym, nil)
```

But I would prefer to keep this as a separate PR / commit / etc, if possible.

Reply:
Syntax here looks nice, thanks for the suggestion!